### PR TITLE
fix: re-verify existing email using torus

### PR DESCRIPTION
**Pull request by sirpy**
# Description
dont check existing email hash when verified via torus
/GoodDollar/GoodDAPP/#3037

### CODE REVIEW

**Comment by sirpy** [comment]
@serdiukov-o-nordwhale can you fix mautic tests?

**Comment by serdiukov-o-nordwhale** [comment]
> @serdiukov-o-nordwhale can you fix mautic tests?

fixed all tests

**Comment by sirpy** [comment]
@omerzam waiting your review

File: src/server/send/__tests__/sendAPI.js [hunk 1]
```diff
@@ -90,6 +52,7 @@ describe('sendAPI', () => {
       .expect(200, { ok: 1 })
   })
 
+  /*
```
Comment (omerzam) [inline src/server/send/__tests__/sendAPI.js:55]: why is this code not removed?

File: src/server/send/__tests__/sendAPI.js [hunk 2]
```diff
@@ -137,5 +100,5 @@ describe('sendAPI', () => {
       })
       .set('Authorization', `Bearer ${token}`)
       .expect(400)
-  })
+  })*/
```
Comment (omerzam) [inline src/server/send/__tests__/sendAPI.js:103]: remove code if not needed

File: src/server/storage/verifier.js [hunk 3]
```diff
@@ -103,7 +100,7 @@ class FacebookVerificationStrategy {
     }
 
     logger.info('FacebookVerifier result:', { emailVerified })
-    userRecord.isEmailConfirmed = userRecord.isEmailConfirmed || emailVerified
+    return { emailVerified, mobileVerified: false }
```
Comment (omerzam) [inline src/server/storage/verifier.js:103]: should this return mobileVerified: false ?

**Review by omerzam — APPROVED** [review]

File: src/server/storage/verifier.js (hunk 3 above)
Comment (serdiukov-o-nordwhale) [reply src/server/storage/verifier.js:103]: @omerzam yes - this is defined in the `verify`'s method contract. it should return `{ emailVerified: boolean, mobileVerified: boolean }` object

**Review by serdiukov-o-nordwhale — APPROVED** [review]

**Pull request merged**